src/test/java/api/support/APITests.java

```diff
@@ -303,18 +311,19 @@ protected void useExampleFixedPolicyCirculationRules()
     useLoanPolicyAsFallback(
       loanPoliciesFixture.canCirculateFixed().getId(),
       requestPoliciesFixture.allowAllRequestPolicy().getId(),
-      noticePoliciesFixture.activeNotice().getId()
+      noticePoliciesFixture.activeNotice().getId(),
+      overdueFinePoliciesFixture.facultyStandard().getId()
     );
   }
 
   protected void useLoanPolicyAsFallback(UUID loanPolicyId, UUID requestPolicyId,
-                                         UUID noticePolicyId)
+                                         UUID noticePolicyId, UUID overdueFinePolicyId)
```

Comment:
Can you rename `useLoanPolicyAsFallback`, for example to `useFallbackPolicies`?

Reply:
Done.